src/Apps/Consign/Routes/ConsignmentInquiry/ConsignmentInquiry.tsx

```diff
@@ -0,0 +1,209 @@
+import { useRouter } from "System/Router/useRouter"
+import createLogger from "Utils/logger"
+import { recaptcha, RecaptchaAction } from "Utils/recaptcha"
+import {
+  ConsignmentInquiryForm,
+  ConsignmentInquiryFormModel,
+} from "Apps/Consign/Routes/ConsignmentInquiry/Components/ConsignmentInquiryForm"
+import { SentConsignmentInquiry } from "@artsy/cohesion/dist/Schema/Events/Consignments"
+import { ConsignmentInquiry_me$data } from "__generated__/ConsignmentInquiry_me.graphql"
+import { createConsignmentInquiry } from "Apps/Consign/Routes/ConsignmentInquiry/utils/createConsignmentInquiry"
+import {
+  consignmentInquiryValidationSchema,
+  validate,
+} from "Apps/Consign/Routes/ConsignmentInquiry/utils/validation"
+import { CreateConsignmentInquiryMutationInput } from "__generated__/createConsignmentInquiryMutation.graphql"
+import { ConsignmentInquiryFormAbandonEditModal } from "Apps/Consign/Routes/ConsignmentInquiry/Components/ConsignmentInquiryFormAbandonEdit"
+import { useState } from "react"
+import { TopContextBar } from "Components/TopContextBar"
+
+const logger = createLogger("ConsignmentInquiry/ConsignmentInquiry.tsx")
+
+const getContactInformationFormInitialValues = (
+  me: ConsignmentInquiry_me$data
+): ConsignmentInquiryFormModel => ({
+  name: me?.name || "",
+  email: me?.email || "",
+  phoneNumber:
+    me?.phoneNumber?.national ||
+    me?.phone ||
+    me?.phoneNumber?.international ||
+    "",
+  phoneNumberCountryCode: me?.phoneNumber?.regionCode || "us",
+  message: "",
+})
+
+export interface ConsignmentInquiryProps {
+  me: ConsignmentInquiry_me$data
+}
+
+export const ConsignmentInquiry: React.FC<ConsignmentInquiryProps> = ({
+  me,
+}) => {
+  const [showModal, setShowModal] = useState(false)
+  const { trackEvent } = useTracking()
+  const { router } = useRouter()
+  const { sendToast } = useToasts()
+  const { relayEnvironment } = useSystemContext()
+  const initialValue = getContactInformationFormInitialValues(me)
+  const initialErrors = validate(
+    initialValue,
+    consignmentInquiryValidationSchema
+  )
+
+  const handleRecaptcha = (action: RecaptchaAction) =>
+    new Promise(resolve => recaptcha(action, resolve))
+
+  const handleSubmit = async ({
+    name,
+    email,
+    phoneNumber,
+    phoneNumberCountryCode,
+    message,
+  }: ConsignmentInquiryFormModel) => {
+    if (!(await handleRecaptcha("consignment_inquiry"))) {
+      return
+    }
+
+    let phoneNumberInternational: string | undefined = undefined
+    if (phoneNumber.trim().length) {
+      phoneNumberInternational = `+${
+        COUNTRY_CODES[phoneNumberCountryCode.toLocaleUpperCase()]
+      } ${phoneNumber.trim()}`
+    }
+
+    const input: CreateConsignmentInquiryMutationInput = {
+      email,
+      message,
+      name,
+      phoneNumber: phoneNumberInternational,
+      userId: me?.internalID,
+    }
+
+    if (relayEnvironment) {
+      try {
+        const consignmentInquiryId = await createConsignmentInquiry(
```

Comment:
**issue:**

In force we use hooks for mutations ([example](https://github.com/artsy/force/blob/main/src/Apps/MyCollection/Routes/EditArtwork/Mutations/useCreateArtwork.ts#L5)). Like this we also don't have to check if the relay environment exists (which should always be true).

Reply:
Thank youuu. Updated.